### ISSUE TITLE
Initialize children docker containers

### ISSUE THREAD

**Issue by smithaustin** [Ross-Alexandra/SENG-371-Project-2#50]
Blocked by [GH-48 Data Partitioning](https://github.com/Ross-Alexandra/SENG-371-Project-2/issues/48)

Child docker containers will need to be initialized with the test algorithm as well as partitioned data. The algorithm will then need to be executed on the input data. If no test algorithm is created, the data should still be input and then some sort of timeout added to represent execution.
### Acceptance Criteria
 * Docker containers are successfully created with test algorithm
 * Partioned data is used properly a input in algorithm
 * Containers all have proper config/libraries etc. (Basically docker config file is working)

**Comment by Ross-Alexandra** [comment]
As of the restructure from the final push, this is no longer needed.
